### ISSUE TITLE
Admin console does not shows up with glassfish7 M1 & M2 release

### ISSUE THREAD

**Issue by ttelang** [eclipse-ee4j/glassfish#23855]
Browser shows a blank page with localhost:4848 or if you navigate through Admin Console link in the Glassfish default page (httlocalhost:8080)

## Environment Details

* GlassFish Version (and build number): Eclipse Glassfish 7 M2 Alpha build 7.0.0-M2-prepare-b262-g9771cb6 
* JDK version: java 17.0.1 2021-10-19 LTS
* OS: MacOS X Big Sur (v11.5.1)
* Database: 
----------

## Problem Description
Admin Console page (localhost:4848) is not loading 

## Steps to reproduce
[//]: # "Describe the bug in detail highlighting current behavior vs expected behavior"
1. Download glassfish .zip file from https://github.com/eclipse-ee4j/glassfish/releases/download/7.0.0-M2/glassfish-7.0.0-M2.zip
2. Unzip the directory and copy it in ~ (home) directory
3. Go to the install directory 
```$ cd ~/glassfish7/bin```
4. Start the server
```$ ./asadmin start-domain domain1```
5. After the above command is successful we see the output 
```
Waiting for domain1 to start .............
Successfully started the domain : domain1
domain  Location: /Users/home/glassfish7/glassfis
h/domains/domain1
Log File: /Users/home/glassfish7/glassfish/domains/domain1/logs/server.log
Admin Port: 4848
Command start-domain executed successfully.
```
6. On opening the following URL localhost:4848 we see a page saying ... 

>> GlassFish Server Administration Console

>> Welcome to Eclipse GlassFish 7.0.0 (build 7.0.0-M2-prepare-b262-g9771cb6 2022-02-24T23:50:57+0000).

>> Status indicator image.	Status: The Admin Console application is loaded.

>> If the browser does not refresh the page automatically please reload the page.

7. Page is stuck here. 
8. On refreshing the page it shows a blank page (completely white). 

The problem is always reproducible 

## Impact of the Issue. 
Unable to use all features of admin console.

**Comment by arjantijms** [comment]
That is "correct". The admin console is not compatible with Faces 4.0.

**Comment by pzygielo** [comment]
Sounds like duplicate of
- #23782

**Comment by arjantijms** [comment]
Indeed, I'll close it as duplicate. Thanks! @ttelang if you want you can add your concerns to https://github.com/eclipse-ee4j/glassfish/issues/23782

**Comment by Oumbarek-Soufiane** [comment]
i have the same issue what is the solution
